Add settings panel for configuring Roborazzi tasks explanation

sanao1006 opened the pull request:
Fixed #493 

- Note on roborazzi task has been added to Roborazzi settings window.
- Add the explanation in README

![image](https://github.com/user-attachments/assets/09250b25-5314-47a7-a836-5f74578c4ee3)

sanao1006 commented on roborazzi-idea-plugin/src/main/kotlin/com/github/takahirom/roborazzi/idea/settings/AppSettingsComponent.kt:
```diff
@@ -3,26 +3,44 @@ package com.github.takahirom.roborazzi.idea.settings
+  private val myMainPanel: JPanel
+  private val imagesPathFromModuleText: JBTextField = JBTextField()
+
+  private val descriptionText = """
+    <html>To enable the display of Roborazzi tasks, please enable<br>
+    <b>Configure all Gradle tasks during Gradle Sync (this can make Gradle Sync slower)</b> in the settings.</html>
+""".trimIndent()
```
- One line does not provide good visibility.
- Depending on the size of the setting window, the text may not be visible.

For these reasons, the text is broken at appropriate positions.

takahirom commented on docs/topics/faq.md:
```diff
@@ -126,3 +126,9 @@ android {
 It is discussed in [this issue](https://github.com/takahirom/roborazzi/issues/272).
 Additionally, it might be worth trying to run your tests with VisualVM to monitor memory usage and identify potential leaks.
+
+### Q: Roborazzi's Gradle task is not displayed.
+
+**A:** It is discussed in [this issue](https://github.com/takahirom/roborazzi/issues/493).  
+To enable the display of Roborazzi tasks, please enable ***Configure all Gradle tasks during Gradle Sync (this can make Gradle Sync slower)*** in the settings.  
+<img src="/docs/images/roborazzi_gradle_task_setting.png" width="800" />
```
Thanks. This FAQ section's question is unclear about what it refers to. It might be helpful to specify that it is related to the Roborazzi IntelliJ Plugin. 👀

takahirom commented on docs/topics/faq.md:
```diff
@@ -126,3 +126,9 @@ android {
+
+**A:** It is discussed in [this issue](https://github.com/takahirom/roborazzi/issues/493).  
+To enable the display of Roborazzi tasks, please enable ***Configure all Gradle tasks during Gradle Sync (this can make Gradle Sync slower)*** in the settings.  
+<img src="/docs/images/roborazzi_gradle_task_setting.png" width="800" />
```
I'm not sure whether this image tag works with our documentation system. 👀

sanao1006 replied:
The images appear to be displayed in the README of the corresponding branch of the fork repository.
However, I would like to adopt your approach because, as you say, it could cause confusion!

![image](https://github.com/user-attachments/assets/d6ca3d24-fddd-483d-a0e7-7ff7156967cf)

sanao1006 replied:
The image for use in README
![image](https://github.com/user-attachments/assets/b9cde28b-ddf5-4a55-a171-ea4685f01416)

takahirom commented on roborazzi-idea-plugin/src/main/kotlin/com/github/takahirom/roborazzi/idea/settings/AppSettingsComponent.kt:
```diff
@@ -3,26 +3,44 @@ package com.github.takahirom.roborazzi.idea.settings
+
+  private val descriptionText = """
+    <html>To enable the display of Roborazzi tasks, please enable<br>
+    <b>Configure all Gradle tasks during Gradle Sync (this can make Gradle Sync slower)</b> in the settings.</html>
```
Could you please clarify the location of the setting? This approach can also be applied to markdown documentation.

```suggestion
    <b>Configure all Gradle tasks during Gradle Sync (this can make Gradle Sync slower)</b> in the Settings | Experimental | Gradle.</html>
```

sanao1006 commented:
@takahirom 

Please review again as I have corrected it!

Also, added automatic line break feature since the text is a little longer.

https://github.com/user-attachments/assets/a9d20e43-1f6c-43cc-8fb5-2f6a97f9c354

takahirom reviewed (approved):
Looks great! Thanks
